.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
+<!-- Add a brief and meaningful description. -->
+
+## Expected Behavior
+<!-- Describe the expected behaviour. -->
+
+## Actual Behavior
+<!-- Describe the current/actual behaviour. -->
+
+## Environment
+
+* Operating system: (E.g RHEL 7.6 )
+* OpenShift version:
+<!-- Run the command `oc version` and add the result here. -->
+* Ansible version:
```

Comment:
Can we remove this? I don't think it makes sense for an operator.

Reply:
Hi @pb82,

Sure. However, could not the gitea-operator installed by ansible scripts? Also, note that the dev/user/contributor is able to remove everything that may not be required. The idea just helps/try to ask all required info.  

Please, let me know if still making sense remove the ansible line and if have anything else that you would like to remove here.